tests/test_maf.py

```diff
@@ -258,6 +258,39 @@ def test_invalid__check_tsa1_tsa2():
         "REFERENCE_ALLELE or all values in TUMOR_SEQ_ALLELE2.\n"
     )
 
+def test_invalid__check_ref_tsa2():
```

Comment:
Could you add the following test cases:
- You don't have one or more of the columns required to perform a comparison and you get **no** error messages
- You don't have the `TUMOR_SEQ_ALLELE1` column and you get an error message
- You don't have the `TUMOR_SEQ_ALLELE1` column and you get **no** error message
- You have all three columns and you get **no** error message

Reply:
Thanks so much for the comments. QQ: do you mean __tsa2__ in the second bullet point? Since tsa1 is optional, we don't throw an error when it's absent. Also, I think the fourth case has been checked in `test_valid__check_tsa1_tsa2()` already. Do I still need to add a test for it?

Reply:
Ah right, I forgot that there's `test_valid__check_tsa1_tsa2` and `test_invalid__check_tsa1_tsa2`. I wonder then if it might be better to change your function name to `_check_ref_tsa1_tsa2` because you're checking all three. And just add your relevant test cases to `test_valid__check_tsa1_tsa2` and `test_invalid__check_tsa1_tsa2` (and rename the test functions?).